bin/curriculum/export/README.md

```diff
@@ -0,0 +1,105 @@
+## export unit progress
+
+This directory contains ruby scripts for exporting unit progress data from the database,
+adding student source code from S3, and filtering the results to exclude PII. the PII filtering
+is done using AWS Comprehend.
+
+## 💰💰 WARNING 💰💰
+
+The PII filtering step can incur substantial costs. Please be sure to double-check your requirements
+before running through these steps.
+
```

Comment:
Is it worth suggesting that folks check what's already in the sharing folder? and is there a good place to write a little summary about what each pull contains? e.g. "This is 1000 samples of student work from all levels in csd3-2024 from x date to y date? If there's "shelf ready" data that's already pulled and pii filtered that meets a data request it would be prudent to just share that data rather than re-pull, right?

Reply:
I love this idea, but I'm lukewarm on adding a readme to every data pull directory. how does this sound:
* internally viewable spreadsheet listing all the info you just mentioned for each data pull
* one README file in s3://cdo-data-sharing/filtered-unit-pii linking to the README in this PR and the spreadsheet

https://docs.google.com/spreadsheets/d/1BiK3a3rlEEto1x9_ITjX7l0CsbhO0WZQrdYNdISWKQA/edit?gid=0#gid=0

thoughts?